firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionTesterApiClientTest.java

```diff
@@ -54,18 +50,32 @@ public class FirebaseAppDistributionTesterApiClientTest {
   private static final String TEST_API_KEY = "AIzaSyabcdefghijklmnopqrstuvwxyz1234567";
   private static final String TEST_APP_ID_1 = "1:123456789:android:abcdef";
   private static final String TEST_PROJECT_ID = "777777777777";
```

Comment:
can you change this to some String. with letters instead of numbers, e.g. `"project-name"` - makes it easier to distinguish from `TEST_PROJECT_NUMBER`

Reply:
Good idea, done!